tests/FSharp.Compiler.ComponentTests/Miscellaneous/FsharpSuiteMigrated.fs

```diff
@@ -0,0 +1,174 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace FSharp.Compiler.ComponentTests.Miscellaneous.FsharpSuiteMigrated
+
+open System
+open System.IO
+open Xunit
+open FSharp.Test
+open FSharp.Test.Compiler
+open FSharp.Test.ScriptHelpers 
+
+
+
+module ScriptRunner = 
+    open Internal.Utilities.Library
+    
+    let private createEngine(args,version) = 
+        let scriptingEnv = getSessionForEval args version
+        scriptingEnv.Eval """
+let errorStringWriter = new System.IO.StringWriter()        
+let oldConsoleError = System.Console.Error
+System.Console.SetError(errorStringWriter)
+let exit (code:int) = 
```

Comment:
I had to do this, since we are running the .fsx files in the same process now.
Some of them have the havit of calling "exit" in order to finish, which I had to shadow using this solution.